worlds/tunic/options.py

```diff
@@ -154,6 +154,29 @@ class ShuffleLadders(Toggle):
     display_name = "Shuffle Ladders"
 
 
+class GrassRandomizer(Toggle):
+    """
+    Turns over 6,000 blades of grass and bushes in the game into checks.
+    """
+    internal_name = "grass_randomizer"
+    display_name = "Grass Randomizer"
+
+
+class GrassFill(Range):
+    """
+    Choose the percentage of your filler/trap items that will be kept local or distributed to other TUNIC players with Grass Randomizer enabled.
+    To keep things balanced, this option must be set to 95% or higher. The host can remove this restriction by turning off the limit_grass_rando setting in host.yaml.
+    This option ignores items placed in your local_items or non_local_items.
+    This option does nothing in single player games or if Grass Randomizer is not enabled.
+    """
+    internal_name = "grass_fill"
+    display_name = "Grass Fill Percent"
+    range_start = 0
+    range_end = 100
+    default = 95
+    visibility = Visibility.template | Visibility.complex_ui | Visibility.spoiler 
```

Comment:
```suggestion
class LocalFill(Range):
    """
    Choose the percentage of your filler/trap items that will be kept local or distributed to other TUNIC players with Grass Randomizer enabled.
    To keep things balanced, this option must be set to 95% or higher. The host can remove this restriction by turning off the limit_grass_rando setting in host.yaml.
    This option ignores items placed in your local_items or non_local_items.
    This option does nothing in single player games or if Grass Randomizer is not enabled.
    """
    internal_name = "local_fill"
    display_name = "Local Fill Percent"
    range_start = 0
    range_end = 100
    default = 95
    visibility = Visibility.template | Visibility.complex_ui | Visibility.spoiler 
```
While working on pot shuffle, I figure we should probably hook into the same local filler option here, so we should probably rename this to local_fill to avoid growing pains later down the line